executor/common.h

```diff
@@ -15,9 +15,13 @@
 #define _GNU_SOURCE
 #endif
 
+#if GOOS_windows
+#include <io.h> //mktemp
+#include <windows.h> //for CreateDirectory
```

Comment:
Stick to 1 style -- either saying "for" or not saying "for". You are adding 2 new lines next to each other and already making them inconsistent.

Reply:
This is needed only for use_temporary_dir, move this include next to use_temporary_dir function. See how other functions use includes.